frontend/packages/design-system/src/themes/FormControlHelperText.ts

```diff
@@ -0,0 +1,13 @@
+import textTheme from './Text';
+
+export default {
+  baseStyle: () => {
+    return {
+      mt: '0.5',
+      _text: {
+        ...textTheme.variants.caption,
+        color: 'neutral.300',
```

Comment:
Why is `color` not part of `textTheme.variants.caption`?

Reply:
Yeah fair point. Maybe there should be a default colour (black), but this looks good for these components.